test/functional/p2p_mutated_blocks.py

```
@@ -0,0 +1,97 @@
+        cmpctblock = HeaderAndShortIDs()
+        cmpctblock.initialize_from_block(block)
+        honest_relayer.send_message(msg_cmpctblock(cmpctblock.to_p2p()))
+
+        # Wait for a `getblocktxn` that attempts to fetch the self-transfer
+        def self_transfer_requested():
+            if not honest_relayer.last_message.get('getblocktxn'):
+                return False
+
+            get_block_txn = honest_relayer.last_message['getblocktxn']
+            return get_block_txn.block_txn_request.blockhash == block.sha256 and \
+                   get_block_txn.block_txn_request.indexes == [1]
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)